schemas/specimen.json

```diff
@@ -51,7 +51,9 @@
       "description": "Specify the tumour staging system used to assess the cancer at the time the tumour specimen was resected. Pathological classification is based on the clinical stage information (acquired before treatment) and supplemented/modified by operative findings and pathological evaluation of the resected specimen.",
       "valueType": "string",
       "meta": {
-        "core": true
+        "core": true,
+        "dependsOn": "sample_registration.tumour_normal_designation",
+        "note": "This field is only required if the specimen is tumour."
```

Comment:
@hknahal this looks good...maybe could say
This field is only required if the specimen is a tumour
or
This field is only required for tumour specimen

Reply:
@kcullion I included it in the "note" field (https://github.com/icgc-argo/argo-dictionary/blob/hn_56/schemas/specimen.json#L56). Are you suggesting it should be in the "description" instead?

Reply:
no sorry was just suggesting to add "a" (so a wording change)

Reply:
or does it make sense to say when specimen is tumour?